nautobot/extras/views.py

```diff
@@ -701,6 +701,56 @@ class DynamicGroupBulkDeleteView(generic.BulkDeleteView):
     table = tables.DynamicGroupTable
 
 
+class ObjectDynamicGroupsView(View):
```

Comment:
Should this be `(generic.ObjectView)` instead?

What permissions are enforced by this view? I'd think you'd at least need `view_dynamicgroup` and `view_<object>` permission - are those checked?